Refer to this link for build results (access rights to CI server needed): 
http://bock.systems:8080/job/hangups_tox/31/

**Build Log**
_last 25 lines_

```
[...truncated 5 lines...]
 > git config remote.origin.url https://github.com/TheInnerCircleO/hangupsbot # timeout=10
Fetching upstream changes from https://github.com/TheInnerCircleO/hangupsbot
 > git --version # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/TheInnerCircleO/hangupsbot +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/19/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/19/merge^{commit} # timeout=10
Checking out Revision 74c0e5761cff07a952bbf24b7268b2a7b598b619 (refs/remotes/origin/pr/19/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 74c0e5761cff07a952bbf24b7268b2a7b598b619
First time build. Skipping changelog.
[workspace] $ /bin/bash /tmp/hudson5260337607730938101.sh
pep8 recreate: /var/lib/jenkins/jobs/hangups_tox/workspace/.tox/pep8
pep8 installdeps: -r/var/lib/jenkins/jobs/hangups_tox/workspace/requirements.txt, -r/var/lib/jenkins/jobs/hangups_tox/workspace/test-requirements.txt
pep8 develop-inst: /var/lib/jenkins/jobs/hangups_tox/workspace
pep8 runtests: PYTHONHASHSEED='1412197223'
pep8 runtests: commands[0] | flake8
./hangupsbot/commands.py:394:80: E501 line too long (84 > 79 characters)
                bot.send_message(event.conv, 'Invalid argument: {}'.format(args[0]))
                                                                               ^
ERROR: InvocationError: '/var/lib/jenkins/jobs/hangups_tox/workspace/.tox/pep8/bin/flake8'
___________________________________ summary ____________________________________
ERROR:   pep8: commands failed
Build step 'Execute shell' marked build as failure
Setting status of 723cf7108705a5dc1a4e2fd196f03857750215e4 to FAILURE with url http://bock.systems:8080/job/hangups_tox/31/ and message: Merged build finished.
```

Oh No...

--- a/hangupsbot/commands.py
+++ b/hangupsbot/commands.py
@@ -675,6 +675,7 @@ def eightball(bot, event, *args):
 
     if not args:
         bot.send_message(event.conv, 'What is your question?')
+        return
 
     answers = [
         "It is certain",

